cli/src/config.rs

```diff
@@ -1457,8 +1455,8 @@ mod tests {
                 ..Default::default()
             },
             wants: &[
-                Want::Existing("config/jj/config.toml"),
                 Want::Existing("config/jj/conf.d"),
+                Want::Existing("config/jj/config.toml"),
```

Comment:
Order matters here so this is a regression---the `config.toml` should come before the `conf.d` directory